src/antelope/stores/history.ts

```diff
@@ -316,4 +503,7 @@ const historyInitialState: HistoryState = {
         current: [],
     },
     __evm_transactions_pagination_data: {},
+    __evm_nft_transfers: {
+        current: new Map(),
+    },
```

Comment:
just to be consistent
```suggestion
    __evm_nft_transfers: {
        [CURRENT_CONTEXT]: new Map(),
    },
```